ops/pebble.py

```diff
@@ -386,11 +386,29 @@ def __repr__(self):
                 ).format(self=self)
 
 
+class Plan:
+    """Represents a Pebble plan (service configuration).
+
+    Currently this only holds the combined services, not the individual
+    layers. The "raw_yaml" attribute is set to a YAML string for access to the
```

Comment:
Similar here as well, we don't need to define what it does not contain.

Reply:
Done.